ambry-api/src/main/java/com.github.ambry/config/NettyConfig.java

```diff
@@ -121,6 +121,15 @@
   @Default("20 * 1024 * 1024")
   public final long nettyMultipartPostMaxSizeBytes;
 
+  /**
+   * If set, use this implementation of {@link com.github.ambry.commons.SSLFactory} to use for the netty HTTP server.
+   * Otherwise, share the factory instance with the router.
+   */
+  public static final String SSL_FACTORY_KEY = "netty.server.ssl.factory";
```

Comment:
move this string to the beginning of this class?

Reply:
+1.  It would be good to have property constants for the others too, but that can be done later.

Reply:
For now, I'll keep the others as is, but I'll look into constant-ifying this and other config classes in the future. Maybe there's a simple way to do it en-masse using intellij refactoring tools.